x/stakeibc/keeper/msg_server_liquid_stake.go

```diff
@@ -91,8 +83,8 @@ func (k msgServer) LiquidStake(goCtx context.Context, msg *types.MsgLiquidStake)
 func (k msgServer) MintStAsset(ctx sdk.Context, sender sdk.AccAddress, amount int64, denom string) error {
```

Comment:
I wonder if we should make this function signature 
```suggestion
func (k msgServer) MintStAsset(ctx sdk.Context, sender sdk.AccAddress, amount uint64, denom string) error {
```

and make `Msg.Amount` a `uint64`

buys us a little bit of safety through types

Reply:
I'm on board with that!

Reply:
might make sense to change all our other `amount`s as well (e.g. RedeemStake, DepositRecord), but I think I'll leave that for another PR